ui/src/flux/components/DatabaseList.tsx

```diff
@@ -49,7 +50,13 @@ class DatabaseList extends PureComponent<Props, State> {
     const {service} = this.props
 
     return databases.map(db => {
-      return <DatabaseListItem db={db} key={db} service={service} />
+      return (
+        <NotificationContext.Consumer key={db}>
+          {({notify}) => (
```

Comment:
interesting to pass notify via context rather than props -- would you mind sharing your reasoning for this?

Reply:
My reasoning for this was i was going to have to send notify down through about 6 components that didn't need notify at all, so using the context was a way to get this function to a few deeply nested children components that actually cared about the function.